tests/OrcidIdTest.php

```diff
@@ -22,4 +22,9 @@ public function testSupportsOrcidsEndingInLowercaseX()
     {
         $this->assertEquals(['0000-0002-1694-233X'], OrcidId::extract('0000-0002-1694-233x'));
     }
+
+    public function testExtractsOrcidsWithTrailingUnicodePunctuation()
+    {
+        $this->assertEquals(['0000-0002-0088-0058'], OrcidId::extract('orcid.org/0000-0002-0088-0058…'));
```

Comment:
Yeah, as it doesn't rely on any character classes affected by Unicode (using only numeric digits), it automatically passes this test. I wanted to put this here to test that it was capable of handling extraction amidst Unicode punctuation already.